<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRouteServiceConfigWidget</class>
 <widget class="QWidget" name="OpenRouteServiceConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>150</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Preference:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="preference"/>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="noMotorways">
     <property name="text">
      <string>Avoid motorways</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="noTollways">
     <property name="text">
      <string>Avoid tollways</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="noFerries">
     <property name="text">
      <string>Avoid ferries</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
